pkg/api/controller.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/treeverse/lakefs/pkg/config"
 	"github.com/treeverse/lakefs/pkg/db"
 	"github.com/treeverse/lakefs/pkg/email"
+	"github.com/treeverse/lakefs/pkg/email/template"
```

Comment:
don't need a package to define a new package for template capability - there are two in go, one for  text and one for html.
if the email body is set to html - you the html template.
you created a method in the controller to post this information - so you can populate it and pass the template.